docs/book/src/topics/multitenancy.md

```diff
@@ -190,3 +190,7 @@ spec:
 ```
 
 For more details on how aad-pod-identity works, please check the guide [here](https://azure.github.io/aad-pod-identity/docs/).
+
+### Workload Identity
+
+Follow this [link](#workload-identity) for workload identity docs.
```

Comment:
```suggestion
Follow this [link](./workload-identity.md) for workload identity docs.
```
AFAICT you need to make this change *and* add it to the ToC in `SUMMARY.md` for the doc to be visible.

Reply:
Done. Thanks